go/state/gostate/go_state.go

```diff
@@ -144,6 +144,14 @@ func (s *GoState) GetStorage(address common.Address, key common.Key) (common.Val
 	return val, s.stateError
 }
 
+func (s *GoState) HasEmptyStorage(address common.Address) (bool, error) {
+	if err := s.stateError; err != nil {
+		return false, err
+	}
+
+	return s.live.HasEmptyStorage(address)
```

Comment:
I guess this would be the place to check for the dedicated not-implemented error to avoid `panic` triggered crashes in Aida.

Reply:
This place wont work for `cpp` state, will it?